Zend/zend_alloc.c

```diff
@@ -396,7 +396,7 @@ static ZEND_COLD ZEND_NORETURN void zend_mm_safe_error(zend_mm_heap *heap,
 }
 
 #ifdef _WIN32
-void
+static void
 stderr_last_error(char *msg)
```

Comment:
```suggestion
static void stderr_last_error(char *msg)
```

Reply:
I noticed the inconsistent coding style, but didn't want to change this unrelated aspect. Maybe time to adopt clang-format?

Reply:
a few lines below it is coded like that, so at least changed lines should use the preffered format

Reply:
Okay, changed the coding style as well.